Jakefile.js

```diff
@@ -1,1076 +0,0 @@
-// This file contains the build logic for the public repo
-
-var fs = require("fs");
-var os = require("os");
-var path = require("path");
-var child_process = require("child_process");
-var Linter = require("tslint");
-var runTestsInParallel = require("./scripts/mocha-parallel").runTestsInParallel;
-
-// Variables
-var compilerDirectory = "src/compiler/";
-var servicesDirectory = "src/services/";
-var serverDirectory = "src/server/";
-var harnessDirectory = "src/harness/";
-var libraryDirectory = "src/lib/";
-var scriptsDirectory = "scripts/";
-var unittestsDirectory = "tests/cases/unittests/";
-var docDirectory = "doc/";
-
-var builtDirectory = "built/";
-var builtLocalDirectory = "built/local/";
-var LKGDirectory = "lib/";
-
-var copyright = "CopyrightNotice.txt";
-var thirdParty = "ThirdPartyNoticeText.txt";

```

Comment:
It was unused. I'll double check if its actually used in the Jakefile and that I forgot something.